### PR TITLE
Update automated a11y tests for website

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

If merged, this PR sets the accessibility automation tests up correctly and resolves issues that were the result of incorrect setup.

CleanShot 2024-04-29 at 15 02 02

### :hammer_and_wrench: Detailed description

- removed a11y-test helper (don't need a specific one)
- added correct config to test-helper file
- added `test:ember-a11y` to package.json

### :link: External links

Jira ticket: [HDS-3214](https://hashicorp.atlassian.net/browse/HDS-3214)
Figma file: [if it applies]

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-3214]: https://hashicorp.atlassian.net/browse/HDS-3214?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/docs/components/dropdown/partials/guidelines/guidelines.md [hunk 1]
```diff
@@ -181,10 +181,10 @@ The height of the ListContainer is automatically determined based on the content
 A Header provides a fixed space at the top of the List. Typically, Headers house a search feature that allows the user to search/filter through the available options in the list. This is great for really long lists when filtering on complex datasets. 
 
 <div class="hds-dropdown__content">
-  <Doc::ListContainer class="hds-dropdown__list">
     <Hds::Dropdown::Header @hasDivider={{true}}>
       <Hds::Form::TextInput::Base @type="search" placeholder="Search" />
     </Hds::Dropdown::Header>
+    <Doc::ListContainer class="hds-dropdown__list">
```
Comment (KristinLBradley) [inline website/docs/components/dropdown/partials/guidelines/guidelines.md:187]: Nit: Could you fix the indenting for the Header above and this ListContainer wrapper?

Comment (KristinLBradley) [reply website/docs/components/dropdown/partials/guidelines/guidelines.md:187]: image

Comment (MelSumner) [reply website/docs/components/dropdown/partials/guidelines/guidelines.md]: ok I think I found/resolved the issue.

**Review by alex-ju — APPROVED** [review]
Looks good! Thanks for aligning it `website` with `showcase`

**Pull request merged**